app/report/views.py

```diff
@@ -93,3 +101,45 @@ def get():
                     "is_visible": is_visible,
                     "results": results
                     }), 200
+
+
+@report.route('/acknowledgment', methods=['POST'])
```

Comment:
Celery task is being triggered twice when you hit this endpoint causing 2 email to be sent. Once on submit and then another a few seconds later. Also if you refresh while still on `https://openrecords-dev-jon.appdev.records.nycnet/report/acknowledgment` it will send the email twice again. We should redirect back to `https://openrecords-dev-jon.appdev.records.nycnet/report/show` after submit.